v2/api/v2/suite_test.go

```diff
@@ -102,12 +108,11 @@ var _ = BeforeSuite(func(done Done) {
 		return nil
 	}).Should(Succeed())
 
-	close(done)
 }, 60)
```

Comment:
This timeout `60` would be meaningless because the 1st argument function of `BeforeSuite()` does not accept a parameter of `done Done`.
I suppose that this timeout and the `done Done` parameter were introduced to prevent hang-up at `DialWithDialer()`.
If so, they should be left unchanged.

To my understanding, `Eventually()` is not purely asynchronous.
It checks its timeout only after `func()` returns.

Some of other `BeforeSuite()`s may be executed without timeout.

Reply:
> this timeout and the `done Done` parameter were introduced to prevent hang-up at `DialWithDialer()`.

Sorry, I'm not sure about this.

Anyway, please do either one of two: remove `60` or revert to use `Done`.

Reply:
I will remove timeout `60`. Asynchronous testing using the done channel is deprecated in Ginkgo V2.
https://github.com/onsi/ginkgo/blob/v2/docs/MIGRATING_TO_V2.md#removed-async-testing